DEVELOP.md

```diff
@@ -39,7 +39,7 @@ Make sure you have the Simulator setup with a working target device. For now you
 
 The `ioTest` task supports the `--tests` flag like other Gradle tasks to filter which tests to run.
 
-Note that for kaluga iOS tests for kaluga run on a background thread, in order to have a properly working Main dispatchers (and align better with Android). How this works is descriped in [`test-utils`](tree/master/test-utils)
+Note that for kaluga iOS tests for kaluga run on a background thread, in order to have a properly working Main dispatchers (and align better with Android). How this works is described in [`test-utils`](test-utils/)
```

Comment:
Maybe you meant "Note that kaluga iOS tests for kaluga run on a background thread" (without first `for`)